src/SourceBuild/content/global.json

```diff
@@ -4,6 +4,7 @@
   },
   "msbuild-sdks": {
     "Microsoft.Build.NoTargets": "3.7.0",
+    "Microsoft.Build.Traversal": "3.4.0",
```

Comment:
FYI I logged https://github.com/dotnet/source-build/issues/4085 to remove the old versions of these package from SBRP.